### PR TITLE
Add support for 6 new languages

### PR DESCRIPTION
**Pull request by jnatalzia**
### This PR will...

Add language support for:
```
Russian
Indonesian
Korean
Thai
Vietnamese
Finnish
```

German file was also updated to fix a typo.
Capitalization changes around "Error Code" were run by and verified by @aliciahurst 

### Why is this Pull Request needed?
We want more support for different languages within the player

### Are there any points in the code the reviewer needs to double check?
None

### Are there any Pull Requests open in other repos which need to be merged with this?
No

#### Addresses Issue(s):

JW8-2669

### CODE REVIEW

**Comment by johnBartos** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  🛠 There are modified src files, but no test changes. Add tests if you're able to.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  🔎 Assign some reviewers or assignees.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  🗿 Set a milestone. It should be the ticket's fix version in JIRA.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ bin-release
 built
 reports
 package-lock.json
+*.DS_Store
```
Comment (karimMourra) [inline .gitignore]: is this a mistake ?

Comment (jnatalzia) [reply .gitignore]: Not accidental, no. These files show up in unstaged for me otherwise, am I missing something?

Comment (robwalch) [reply .gitignore]: @jnatalzia This is fine, but it's good to add that to your global git config/ignore settings https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Comment (jnatalzia) [reply .gitignore]: 👍

**Pull request merged**